### PR TITLE
Podcast container test

### PR DESCRIPTION
**Pull request by tomrf1**
## What does this change?
_(The 1st commit is just removing the previous test)_
Adds a design test to the uk network front.
We are testing 3 variants of a podcast container and measuring click-through.

In general, clicking on the container takes a user to the episode page. We track both the impression and the click.

Variant A is almost identical to the World Cup Daily test we did previously.
Variants B/C also have links to the Audio Long Read tag page and apple/google/spotify. These clicks will be tracked using the `data-link-name` rather than the ab test frameworks tracking.

## Screenshots
(there are many more breakpoints that I haven't screenshotted)

##### Variant A
picture 2

##### Variant B
picture 1

##### Variant C
picture 3

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: static/src/stylesheets/module/journalism/_podcast-container-c.scss [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+.podcast-container-c__thrasher {
+    background-color: $brightness-20;
+
+    .fc-container__header__title {
+        color: $brightness-100;
+    }
+}
+
+.podcast-container-c__main {
+    display: grid;
```
Comment (ajwl) [inline static/src/stylesheets/module/journalism/_podcast-container-c.scss]: Did frontend allow you to use `grid`? 
Do we have to do some fallback?

File: static/src/javascripts/projects/common/modules/experiments/tests/podcast-container.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    author: 'Tom Forbes',
+    description: 'Test designs for a /uk podcasts container',
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: 'Measure click-through across the variants',
+    audienceCriteria: '',
+    showForSensitive: true,
+    canRun() {
+        return (
+            config.page.pageId === 'uk' && window.CSS.supports('display: grid')
```
Comment (ajwl) [inline static/src/javascripts/projects/common/modules/experiments/tests/podcast-container.js]: Ah - i see - cunning: `window.CSS.supports('display: grid')`

**Comment by ajwl** [comment]
👍 looks good!

File: static/src/javascripts/projects/common/modules/experiments/tests/podcast-container.js (hunk 2 above)
Comment (tomrf1) [reply static/src/javascripts/projects/common/modules/experiments/tests/podcast-container.js]: yep, older browsers will not see the test - though support for grid seems pretty good now

**Comment by ajwl** [comment]
FTR: to preview the test - 
http://localhost:9000/uk#ab-PodcastContainer=b

**Pull request merged**